Dockerfile-prod-base.in

```diff
@@ -131,4 +131,6 @@ RUN /.install-deps.sh
 RUN \
     rm -f /.install-deps.sh && \
     apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* && \
+    groupadd -f -g 110 render && \
```

Comment:
@neodesys can we add a comment here to explain for others why we need this group?